stacks/ring.go

```diff
@@ -14,7 +14,11 @@ type ring struct {
 	end int
 }
 
+// Write writes b[] into the ring buffer at the current write position (r.end)  - wrapping if needed, and increasing r.end (possibly wrapping)
 func (r *ring) Write(b []byte) (int, error) {
+
+	//println("wrote ",string(b),"into tx ring buff") //@@@REMOVES
```

Comment:
Leftover?

Reply:
yep - sorry - not sure on the workflow here - if i just fix and commit - does it create a whole lot of noise again ?

Reply:
Don't worry, GitHub is clever, you can go either way: make changes, create new commit and push it or make changes, squash & amend commits to single one and then push.

Some projects favor one over the other - personally I prefer the second approach as it keeps git history cleaner :slightly_smiling_face: